app/components/animated_number/index.tsx

```diff
@@ -26,43 +26,43 @@ const AnimatedNumber = ({
     animateToNumber,
     animationDuration,
     fontStyle,
-    easing,
+    easing = 1.2,
 }: Props) => {
     const prevNumber = usePrevious(animateToNumber);
     const animateToNumberString = String(Math.abs(animateToNumber));
     const prevNumberString = String(Math.abs(prevNumber));
 
-    const numberStringToDigitsArray = Array.from(animateToNumberString, Number);
-    const prevNumberersArr = Array.from(prevNumberString, Number);
-
     const [numberHeight, setNumberHeight] = React.useState(0);
-    const animations = useMemo(() => numberStringToDigitsArray.map((__, index) => {
-        if (typeof prevNumberersArr[index] !== 'number') {
-            return new Animated.Value(0);
+    const animations = useMemo(() => {
+        if (numberHeight === 0) {
+            return [];
```

Comment:
Q: would it be safer to return 

```ts
    return numberStringToDigitsArray.fill(new Animated.Value(0));
```

just to protect from **future** regression?

Could transform.translateY: undefined

```ts
                              translateY: animations[index],
```

cause any issue if index is out of bounds due to animations being empty.

Reply:
Yes, but I think `numberStringToDigitsArray ` is a `number` array and cannot be filled with `Value` type. Maybe `map` can work here to return the new array filled with `Animated.Value(0)`.